test/compilable/ddoc_markdown_links.d

```diff
@@ -0,0 +1,33 @@
+// PERMUTE_ARGS:
+// REQUIRED_ARGS: -D -Dd${RESULTS_DIR}/compilable -o- -preview=markdown
+// POST_SCRIPT: compilable/extra-files/ddocAny-postscript.sh
+
+/++
+# Links
+
+[unused reference]: https://nowhere.com
+
+A link to [Object].
+
+A link to [the base object][Object].
+
+Not a link because it's an associative array: int[Object].
+
+An inline link to [the D homepage](https://dlang.org).
+
+A reference link to [the D homepage][d site].
```

Comment:
Does this reference the above link?

Reply:
No, it matches the `[D Site]` reference on line 29. The text in the first set of brackets is display-only and isn't used to match references.

Reply:
Ah, it's the other way around.